manifests/init.pp

```diff
@@ -47,7 +51,7 @@
 
   case $::osfamily {
     'RedHat': {
-      case $::lsbmajdistrelease {
+      case $::operatingsystemmajrelease {
```

Comment:
This can work in this use case since it is under RedHat. If used elsewhere it will break as Suse does not support the new fact.